src/utils.js

```diff
@@ -330,4 +330,14 @@ function shouldNavigate(event) {
   );
 }
 
-export { stripSlashes, pick, match, resolve, combinePaths, shouldNavigate };
+function hostMatches(anchor) {
+  const host = location.host
+  return (
+    anchor.host == host ||
+    // svelte seems to kill anchor.host value in ie11, so fall back to checking href
+    anchor.href.indexOf(`https://${host}`) == 0 ||
+    anchor.href.indexOf(`http://${host}`) == 0
+  )
```

Comment:
This is a great fix! I will merge this if these `==` are turned into `===`. 🤓

Reply:
Will do, thanks!

Reply:
Done